Directory.Build.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <!-- Begin Orleans: Without these lines the project won't build properly -->
+  <!-- Set path to ClientGenerator.exe -->
+  <PropertyGroup Condition=" '$(OrleansBuildTimeCodeGen)' == 'true' ">
+    <OrleansReferencesBase>$(ProjectDir)..\Bootstrap\</OrleansReferencesBase>
```

Comment:
Probably all flavors do the same, not sure, but wouldn't it make sense to have BuildFlavor and Configuration part of the path for the Bootstrap.
Also, instead of ProjectDir, we probably want to use the path to this msbuild script, which will not change from project to project, even if they are nested differently.